website/src/components/views/compareToBaseline/SelectBaseline.astro

```diff
@@ -0,0 +1,18 @@
+---
+import { PageHeadline } from '../../../styles/containers/PageHeadline';
+---
+
+<div class='mx-4 mt-20 flex min-h-64 flex-col items-center'>
+    <PageHeadline>Analyze a variant compared to a baseline</PageHeadline>
+    <div class='max-w-xl'>
+        <p class='mb-2'>To proceed, please select a variant filter.</p>
+        <p>
+            If no baseline is selected all sequences will be used as a baseline for comparing against sequences with a
+            specified variant.{' '}
+            Note that the dataset filter is applied to the entire dataset including both the baseline and variant subsets.{
+                ' '
+            }
```

Comment:
Those whitespaces look funny. Do we need them at all?

Reply:
Ah true! Good catch - they aren't needed in astro - prettier formatted them weirdly - I have now removed :-)